src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -269,6 +269,14 @@ export type UI = {
     extended?: boolean | undefined,
   ) => void;
   readonly close: () => void;
+  readonly setPosition: (options: SetUIPositionOptions) => void;
+};
+
+export type SetUIPositionOptions = {
```

Comment:
```suggestion
export type WidgetPositionOptions = {
```

Reply:
I just realize we have widget exposed directly so i updated the types base on type `WidgetLocation`.